### ISSUE TITLE
Inverted proportion in Masonry mode

### ISSUE THREAD

**Issue by luc-pitipuis** [Ecodev/natural-gallery-js#32]
Hi, in Masonry mode, a portrait image has a landscape miniature on the gallery, and vice versa.

I tracked down the issue, and it's there:

https://github.com/Ecodev/natural-gallery-js/blob/b7d4cad4ecfab557eda51159f8c2ed8b8385891c/app/js/galleries/Masonry.ts#L155

it should be
```
item.height = item.width * item.model.enlargedHeight / item.model.enlargedWidth;
```

**Comment by sambaptista** [comment]
Thanks, I'll check this.

**Comment by sambaptista** [comment]
Just published 4.1.1 that fix it.
